### DISCUSSION TITLE
Error during synchronization between servers: compreface-core executing GET http://compreface-core:3000/status

### DISCUSSION THREAD

**Discussion by tcpipchip** [exadel-inc/CompreFace#1150] [General]
![image](https://github.com/exadel-inc/CompreFace/assets/11488460/b7967b7e-3a9b-4fa9-81db-17953ebaa042)

Do you have idea of problem ?

Running on Virtual Box...maybe is too slow....

**Comment by pospielov** [comment]
This error usually means that compreface-core container didn't start well. 
Usually, it happens if the hardware doesn't support something. 
Are you sure Virtual Box has access to AVX CPU instructions?

**Reply by pospielov** [reply]
Ok, I see you already solved it. 
https://github.com/exadel-inc/CompreFace/issues/1149

**Comment by tcpipchip** [comment]
https://nina-b302-wifi-arduino.blogspot.com/2023/08/espcam32-compreface.html
:)
